src/IntegratorFactory.php

```diff
@@ -450,6 +462,7 @@ public function getManifestExecutorStrategies(): array
             $this->createWireGlueRelationshipManifestStrategy(),
             $this->createUnwireGlueRelationshipManifestStrategy(),
             $this->createExecuteConsoleManifestStrategy(),
+            $this->createNavigationManifestStrategy(),
```

Comment:
Looking on the other strategies we usually have wire and unwire.

Because of strict deadline we can skip it. But let's first check how quick it could be added.

Reply:
I think this is a pretty big task. It's better to put it into the backlog.